.docker/autoline/frontend.Dockerfile

```
@@ -0,0 +1,26 @@
+FROM node:16.16.0-alpine as frontend-build
+
+WORKDIR /app
+
+COPY ./package*.json ./
```

We need to copy `yarn.lock`
```suggestion
COPY ./package.json ./
COPY ./yarn.lock ./
```